chaincode-labs/chaincode-podcast/privacy-and-robustness-in-lightning.md

```diff
@@ -331,112 +353,112 @@ When it's my turn, by definition, you haven't got anything in flight.
 And it turns out that supporting both is actually pretty easy.
 
-Speaker 2: 00:16:10
+Adam Jonas: 00:16:10
 
-So you're talking about when we were talking about L2, you're talking about changes and you're sort of casually mentioning the soft fork piece of it.
-I mean, there's not going well, there's Greg Sanders will be there and he's obviously thinking about this, but you don't really have the people at the table that are actually doing like would be pulling the soft fork together that are that really understand the lightning case, I guess.
-Maybe AJ, maybe InstaGibs.
+When we were talking about  Eltoo, you're talking about changes and you're sort of casually mentioning the soft fork piece of it.
+I mean, there's not going well, there's Greg Sanders will be there and he's obviously thinking about this, but you don't really have the people at the table that are actually doing like would be pulling the soft fork together that really understand the lightning case, I guess.
+Maybe AJ, maybe Instagibbs.
 The list gets very short after that.
 So, like, how are you able to communicate those needs down to the lower layer?
 
-Speaker 0: 00:16:47
+Rusty Russell: 00:16:47
 
-Well, I think the any prep out cases have been pretty strong for a long time.
+Well, I think the `ANYPREVOUT` cases have been pretty strong for a long time.
 The question's always been, do we want other soft fork as well?
 
 ## CTV
 
-Speaker 0: 00:16:54
+Rusty Russell: 00:16:54
 
-And people are like, oh, if we had obstacle the check sick from stack, we can also do it.
+And people are like, oh, if we had `OP_CTV` and `OP_CHECKSIGFROMSTACK`, we can also do it.
 Sure.
 I can probably build a car out of matchsticks and snot, but I don't want to.
 And don't make me do it.
 
-Speaker 2: 00:17:09
+Adam Jonas: 00:17:09
 
 Could you do that?
 
-Speaker 0: 00:17:12
+Rusty Russell: 00:17:12
 
 A theoretical engineer probably could do that.
 I could not do that.
 I'm not crafty at all.
 But, okay.
 That is probably an aside.
 We don't need to go down.
-But, you know, check sync from stack is like, oh, it's cool that you can even make that work.
+But,`CHECKSIG` from stack is like, it's cool that you can even make that work.
 But it is a terrible basis for just about anything.
 And it's more like a game we could play as far as what's the most obscure way we could do some kind of introspection.
-And the answer would be, you know, check-sig from stack.
-So any prep-out is a – and the proposals that are in that cluster, and there are a few, are pretty clean, pretty well understood.
-AJ would definitely be the person unfortunately he's not here in New York he's kind of you know done a fair bit of travel recently he was like no I'm not going.
-So I'm hoping that while we're in New York he is in his mountain lair somewhere coming up with a scheme to activate any prep out.
+And the answer would be, `CHECKSIG` from stack.
```

Comment:
Here also

Reply:
Noted

Reply:
you didn't make this change, please see this commit to understand what we were asking for: https://github.com/bitcointranscripts/bitcointranscripts/pull/482/commits/9eda4c9e97f7ff98505827591be656f81a2833ac